### ISSUE TITLE
ORCA 6 Parsing Issues

### ISSUE THREAD

**Issue by spencer-leger** [QChASM/SEQCROW#18]
Hello, this may already be on your radar, but in case it is not, there was recently the ORCA 6 release. In the new release, some output has been changed, which results in SEQCROW not being able to parse the information correctly. Most functionality is still there, but from what I've tested, I've noted these issues:

1) Opening frequency calculations and visualizing normal modes when !UseSym is used
2) Viewing NMR spectra
3) Viewing UV/Vis spectra

This is by no means a complete list, but is what I noticed so far. These same calculations from ORCA 5.0.4 open correctly.

Thanks in advance!

**Comment by ajs99778** [comment]
I will have to check !UseSym, but as far as I'm aware the latest version of SEQCROW (1.8.10) is able to parse NMR and UV/vis data from ORCA 6 output. If ChimeraX has not prompted you to update, you can do it manually by going to Tools &rarr; More Tools... on the ChimeraX menu, navigating to the SEQCROW page on the toolshed, and clicking the upgrade button. 

If you still experience issues with NMR and UV/vis output after updating, it would be helpful if you send me an example file.

**Comment by spencer-leger** [comment]
Interesting, I only had v1.8.8, but it did not show the update in Tools > Utilities > Updates. Thanks for that tip, I've updated and most of the issues are gone. The use of `!UseSym` still leads to a parsing issue, at least for frequency calculations. UV/Vis spectra work fine with and without frequency calculations. However, jobs with `!NMR freq` only seem to parse the NMR data and not the frequency data. I've attached a simple example of an output file that shows this (note that I had to upload it as a LOG file).

[formaldehyde_nmr_freq.out.log](https://github.com/user-attachments/files/16598580/formaldehyde_nmr_freq.out.log)

**Comment by ajs99778** [comment]
Thanks for providing an output file. I've uploaded a patch that looks like it works to github. Feel free to try it out. Instructions for installing from github are here: https://github.com/QChASM/SEQCROW?tab=readme-ov-file#github. I'll do a little bit more testing before uploading it to the toolshed. 

For NMR jobs, it looks like the line we were using to figure out when we're done parsing NMR data doesn't get printed in ORCA 6. So nothing after NMR was getting parsed. 

For frequency calculations with symmetry, some changes had to be made to the normal mode displacement parser. Previously, we assumed it would print no more than 6 displacements per line. With `!UseSym`, it looks like ORCA will print more. We also had to adjust for the symmetry labels, which should now appear on the normal mode tool if they are present in the output file.
